spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java

```diff
@@ -196,65 +202,70 @@ void shouldInstantiateMultiRabbitConnectionFactoryWrapperWithDefaultConnection()
         final MultiRabbitConnectionFactoryWrapper externalWrapper = new MultiRabbitConnectionFactoryWrapper();
 
         when(springFactoryCreator.rabbitConnectionFactory(rabbitProperties, resourceLoader, credentialsProvider,
-                credentialsRefreshService, connectionNameStrategy)).thenReturn(new CachingConnectionFactory());
+                credentialsRefreshService, connectionNameStrategy,
+                connectionFactoryCustomizers)).thenReturn(new CachingConnectionFactory());
```

Comment:
you can use the previous line as much as possible, and leave only .thenReturn(...); at the new line.